includes/classes/admin/ajax/handlers.php

```diff
@@ -404,6 +405,46 @@ public function gc_dismiss_notice_cb() {
 		wp_send_json_error();
 	}
 
+	/**
+	 * Ajax callback when getting Component subfields for the mapping UI
+	 *
+	 * @since  3.2.20
+	 *
+	 * @return void
+	 */
+	public function gc_component_subfields_cb() {
+		if ( ! $this->_post_val( 'subfields_data' ) ) {
+			wp_send_json_error("one");
+		}
+		$data = $this->_post_val( 'subfields_data' );
+
+		// If ACF Field group
+		if ( $data['name'] && str_contains($data['name'],"group") ){
+			$field_group = acf_get_fields($data['name']);
+			// If Repeater with sub fields
+			if($field_group){
+				$success_data = array(
```

Comment:
Are we able to use the new array syntax as using `array()` is super old and in my opinion should be something we don't do anymore and the `[]` syntax is supported for quite a while now.

Reply:
Hey @CWDN @HughbertD 

We did not end up making this change as the bulk of the plugin is currently using the "array()" syntax. Making this update would fall outside of the features we worked on from a consistency standpoint. Let us know if you have any questions on that!

Reply:
Hey @pat-kavanagh 

Definitely see where Chris is coming from! 

But I don't think that's too much of an issue, we can run a linter on our side in the future across the plugin and do this in one sweep